### PR TITLE
Add Ariba connector

### PR DESCRIPTION
**Pull request by indrapaul24**

### CODE REVIEW

File: connectors/sap-ariba/README.md [hunk 1]
```diff
@@ -1,5 +1,170 @@
...
-
+difficulty_level: INTERMEDIATE
+time_in_minutes: 60
+accreditations:
+- DEFAULT
 ---
+
+# Introduction
+
+SAP Ariba is a comprehensive cloud-based solution for procurement and supply chain management. This guide covers the access needs that are required to establish a stable connection to your Ariba Instance, authenticate with SAP Ariba’s APIs and test it successfully in Creator Studio.
```
Comment (somu-a) [inline connectors/sap-ariba/README.md:12]: Add a comma after "SAP Ariba's APIs", so it should be This guide covers the access needs that are required to establish a stable connection to your Ariba Instance, authenticate with SAP Ariba’s APIs**,** and test it successfully in Creator Studio.

**Review by somu-a — COMMENTED** [review]
One quick comma fix

**Review by somu-a — APPROVED** [review]
Approved.

**Review by ajaymerchia — APPROVED** [review]
Some partially redacted secrets can be fully redacted

**Pull request merged**